src/classes/BGE_ConfigurationWizard_CTRL.cls

```diff
@@ -233,6 +233,20 @@ public with sharing class BGE_ConfigurationWizard_CTRL {
         return dfrs;
     }
 
+/*******************************************************************************************************
+    * @description Returns the corresponding Lightning field type from an Apex field type
+    */
+    public static String getLightningFieldType(String apexFieldType) {
```

Comment:
If this method also supports lightning components, should we move it into UTIL_LightningComponetSupport alongside getLightningDataType()?